gcc/rust/resolve/rust-ast-resolve-type.cc

```diff
@@ -100,7 +100,7 @@ ResolveRelativeTypePath::go (AST::TypePath &path, NodeId &resolved_node_id)
 	{
 	  // error[E0433]: failed to resolve: `self` in paths can only be used
 	  // in start position
```

Comment:
```suggestion
	  // TODO: error[E0433]: failed to resolve: `self` in paths can only be used
	  // in start position
```
we need to handle that specific suberror as well - the error code is the same, but the message changes: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=182946aafa12c0b0e3dce0cda8851faa